plugins/CMakeLists.txt

```
@@ -189,6 +189,17 @@ if(BUILD_SKELETON)
     add_subdirectory(skeleton)
 endif()
 
+macro(SUBDIRLIST result subdir)
+    file(GLOB children ABSOLUTE ${subdir}/ ${subdir}/*/)
+    set(dirlist "")
+    foreach(child ${children})
+        if(IS_DIRECTORY ${child})
+            file(RELATIVE_PATH child ${CMAKE_CURRENT_SOURCE_DIR}/${subdir} ${child})
+            list(APPEND dirlist ${child})
+        endif()
+    endforeach()
+    SET(${result} ${dirlist})
```

our convention is lowercase for function/macro calls
```suggestion
    set(${result} ${dirlist})
```